src/foundry/client/apps/forms/journal-page-sheet.d.ts

```diff
@@ -0,0 +1,292 @@
+import type { EditorView } from "prosemirror-view";
+import type Showdown from "showdown";
+import type { Editor } from "tinymce";
+import type { JOURNAL_ENTRY_PAGE_FORMATS } from "../../../common/constants.mjs.js";
+import type { CONST } from "../../../common/module.mjs.js";
+
+declare global {
+  /**
+   * The Application responsible for displaying and editing a single {@link JournalEntryPage} document.
+   *
+   * @typeParam TOptions - The type of the options object
+   */
+  class JournalPageSheet<TOptions extends JournalPageSheet.Options = JournalPageSheet.Options> extends DocumentSheet<
```

Comment:
nit: try to use the same name for a generic parameter if there's a superclass, `Document` calls it `Options`. Plus I don't think the repo should adhere to hungarian case.

Reply:
Turns out it was to disambiguate `TOptions` from `Options`; my oversight.

Reply:
Instead of introducing hungarian notation to the repo, please name it `ConcreteOptions` instead.